kedro-datasets/RELEASE.md

```diff
@@ -44,13 +44,15 @@ Many thanks to the following Kedroids for contributing PRs to this release:
 * Fixed bug with loading models saved with `TensorFlowModelDataset`.
 * Make dataset parameters keyword-only.
 * Correct pandas-gbq as py311 dependency
+* Fixed bug with loading datasets from Hugging Face.
 
 ## Community contributions
 Many thanks to the following Kedroids for contributing PRs to this release:
 * [Edouard59](https://github.com/Edouard59)
 * [Miguel Rodriguez Gutierrez](https://github.com/MigQ2)
 * [felixscherz](https://github.com/felixscherz)
 * [Onur Kuru](https://github.com/kuruonur1)
+* [Eduardo Romero Lopez](https://github.com/eromerobilbomatica)
```

Comment:
@eromerobilbomatica These changes should be under the "Upcoming Release" heading at the top of this page!